content/actions/creating-actions/publishing-actions-in-github-marketplace.md

```diff
@@ -41,7 +41,11 @@ To draft a new release and publish the action to {% data variables.product.prodn
 1. Navigate to the action metadata file in your repository (`action.yml` or `action.yaml`), and you'll see a banner to publish the action to {% data variables.product.prodname_marketplace %}. Click **Draft a release**.
 1. Under "Release Action", select **Publish this Action to the {% data variables.product.prodname_marketplace %}**.
 
-   If you can't select the checkbox, you must first click the link to read and accept the {% data variables.product.prodname_marketplace %} Developer Agreement.
+   {% note %}
+   
+   The "Publish" checkbox is disabled if the user or organization that owns the repository has not yet accepted the {% data variables.product.prodname_marketplace %} Developer Agreement. If you own the repository or are an organization owner, a link to "accept the GitHub Marketplace Developer Agreement" is displayed. Click this link and accept the agreement to continue. If there is no link, send the organization owner a link to this "Release Action" page and ask them to accept the agreement.
```

Comment:
Some ideas of how to shorten if you want, all totally optional (except adding the **Note** to start).

```suggestion
   **Note**: The "Publish" checkbox is disabled if the account that owns the repository has not yet accepted the {% data variables.product.prodname_marketplace %} Developer Agreement. If you own the repository or are an organization owner, click the link to "accept the GitHub Marketplace Developer Agreement", then accept the agreement. If there is no link, send the organization owner a link to this "Release Action" page and ask them to accept the agreement.
```